saleor/graphql/core/types/filter_input.py

```diff
@@ -0,0 +1,63 @@
+import six
+from graphene import InputField, InputObjectType
+from graphene.types.inputobjecttype import InputObjectTypeOptions
+from graphene.types.utils import yank_fields_from_attrs
+from graphene_django.filter.utils import get_filterset_class
+
+from .converter import convert_form_field
+
+
+class FilterInputObjectType(InputObjectType):
+    """Class for storing and serving django-filtres as graphQL input.
+    FilterSet class which inherits from django-filters.FilterSet should be
+    provided with using fitlerset_class argument."""
+    @classmethod
+    def __init_subclass_with_meta__(
+            cls, _meta=None, model=None, filterset_class=None,
+            fields=None, **options):
+        cls.custom_filterset_class = filterset_class
+        cls.filterset_class = None
+        cls.fields = fields
+        cls.model = model
+
+        if not _meta:
+            _meta = InputObjectTypeOptions(cls)
+
+        fields = cls.get_filtering_args_from_filterset()
+        fields = yank_fields_from_attrs(fields, _as=InputField)
+        if _meta.fields:
+            _meta.fields.update(fields)
+        else:
+            _meta.fields = fields
+
+        super().__init_subclass_with_meta__(_meta=_meta, **options)
+
+    @classmethod
+    def get_filtering_args_from_filterset(cls):
+        """ Inspect a FilterSet and produce the arguments to pass to
+            a Graphene Field. These arguments will be available to
+            filter against in the GraphQL
```

Comment:
I am not sure if I understand this sentence, what do you mean with `in GraphQL`?

Reply:
It was taken directly from graphene_django.
I assume that it means that it takes all discovered filterset's fields and convert them to graphQL's types. Then, all these arguments will be visible in the schema.